lib/helpers.js

```diff
@@ -60,6 +60,24 @@ exports.checkPropertyName = function(name) {
 };
 
 /**
+ * Validate remote method name. Allows `.` in method name for prototype
+ * @param {String} name The user input
+ * @returns {String|Boolean}
+ */
+exports.validateRemoteMethodName = function(name) {
+  if (name.match(/\./) && !name.match(/^prototype\.([^.]*)$/)) {
+    return 'Name cannot contain special characters . ' +
```

Comment:
Nitpick: I think "special characters ." is not grammatically correct. How about the following?

```js
'Name cannot contain "." characters except the dot in "prototype." prefix'
```